docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -130,11 +104,11 @@ In this tutorial, you created a .NET console application. In the next tutorial,
 
 ::: zone pivot="dotnet-7-0"
```

Comment:
We should either keep  .NET 6 and 7 sections as they were with no changes or do in them any edits we're doing now for .NET 8. We could consider omitting them and going with .NET 8 only for this update.